### ISSUE TITLE
scss-lint:disable all not working - linter having problems with commented out html

### ISSUE THREAD

**Issue by afridley** [adamwalzer/vscode-scss-lint#17]
I have a scss file with a large section of code commented out at the top.
looks something like

```
/* doc
----
Title
name
----
```html example
<div class="example">
</div>

### Example

*/

```
For some reason when this is present the linter shows errors on the scss below that don't exist in random places almost as if everything has been shifted.

Ex. background-position: center center; 
gets the error StringQuotes prefer double quoted strings.

If I delete the top commented out documentation everything works fine.

I would like to use 
scss-lint:disable all 
then
scss-lint:enable all 
to ignore this section but I am still seeing the wrong errors.

**Comment by adamwalzer** [comment]
@afridley Thanks for reporting the issue. I think I'll be able to take care of this one. Can I get more information about your system? Windows? Mac? Version of scss-lint?

**Comment by adamwalzer** [comment]
This issue should be fixed for macs now. I'm not sure if it's still occurring on windows.

**Comment by afridley** [comment]
Works for me! I am on a Mac though so I can't vouch for windows.

**Comment by adamwalzer** [comment]
Great to hear. I'll try to test this in windows before closing the issue though. Thanks!

**Comment by adamwalzer** [comment]
Works on windows for me. Closing the ticket :-)
